tests/local_test_transformer.py

```diff
@@ -0,0 +1,550 @@
+# Copyright 2023 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import unittest
+
+from dwave.plugins.sklearn.transformers import SelectFromQuadraticModel
+
+import numpy as np
+import pandas as pd
+import logging
+
+
+class TestTransformer(unittest.TestCase):
+    def __init__(self, methodName: str = None) -> None:
```

Comment:
As @randomir pointed out elsewhere, you should use [setUpClass](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass)